bokehjs/test/unit/models/tools/gestures/tap_tool.ts

```diff
@@ -42,67 +42,155 @@ describe("TapTool", () => {
     ui_event_bus._trigger(ui_event_bus.doubletap, event, new Event("mousemove"))
   }
 
-  describe("should support 'tap' gesture", () => {
+  describe("should support 'tap' gesture with behavior 'select'", () => {
     it("and trigger on 'tap' event", async () => {
       let called = false
-      const callback = {execute() { called = true }}
+      let behavior = "select"
+      const callback = {execute() { called = true, behavior = "select" }}
```

Comment:
This is not currently testing the *passed-in value*. It will need to be something like:
```suggestion
      let behavior = null
      const callback = {execute(_cb_ob, data) { called = true, behavior = data.behavior }}
```
That should verify that the value that the callback *receives* is correct. (Similar changes below)